src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -969,9 +1026,15 @@ abstract class ManagedStrategy(
         methodId: Int,
         params: Array<Any?>
     ) {
-        val guarantee = runInIgnoredSection {
+        var guarantee: ManagedGuaranteeType? = null
+
+        runInIgnoredSection {
+            // process method effect on the static memory snapshot
+            processMethodEffectOnStaticSnapshot(owner, params)
```

Comment:
What does this mean?

Reply:
You mean the name of the method or what it does?
This method checks what kind of function was called (Unsafe API, VarHandle API, Java AFU) and updates static snapshot accordingly